### PR TITLE
Split jobs

### PR DESCRIPTION
**Pull request by emteknetnz**

### CODE REVIEW

File: config/jobs/recipe-core-main.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ jobs:
     - php: 7.1
       env:
         - DB=MYSQL
-        - REQUIRE_RECIPE_CORE=4.5.x-dev
```
Comment (dnsl48) [inline config/jobs/recipe-core-main.yml]: should it be `REQUIRE_INSTALLER`?

**Review by dnsl48 — CHANGES_REQUESTED** [review]

File: config/jobs/recipe-core-main.yml (hunk 1 above)
Comment (emteknetnz) [reply config/jobs/recipe-core-main.yml]: nope, should be REQUIRE_RECIPE_CORE

File: config/jobs/recipe-core.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ jobs:
     - php: 7.1
       env:
         - DB=MYSQL
-        - REQUIRE_RECIPE_CORE=4.5.x-dev
```
Comment (dnsl48) [inline config/jobs/recipe-core.yml]: typo?

**Review by dnsl48 — CHANGES_REQUESTED** [review]

File: config/jobs/recipe-core-main.yml (hunk 1 above)
Comment (dnsl48) [reply config/jobs/recipe-core-main.yml]: haha, I think I reviewed a different thing there..

**Review by dnsl48 — APPROVED** [review]

File: config/jobs/recipe-core.yml (hunk 2 above)
Comment (emteknetnz) [reply config/jobs/recipe-core.yml]: nope

**Pull request merged**